setup_connection.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Thanks @kikkomep! Apologies for the delay. One question, this bash script is used by the container and helm charts to setup connection parameters. Changing the route for this implies that this needs to be changed accordingly in the chart and possibly the container. A change like this would normally imply some backwards incompatibility that needs to be handled through versioning and documentation. Is is necessary to remove this setup mechanism? Have you made adequate changes at the other side? Thanks!

Reply:
Hi @pcm32 ! 
I've pushed the required changes to the containers and helm charts via the following PRs:
* https://github.com/phnmnl/container-phenomenal-portal/pull/5
* https://github.com/phnmnl/container-portal-metadata-backend/pull/1
* https://github.com/phnmnl/helm-charts/pull/4

Reply:
Great, thanks!